src/sentry/seer/anomaly_detection/utils.py

```diff
@@ -102,28 +129,31 @@ def format_historical_data(data: SnubaTSResult, dataset: Any) -> list[TimeSeries
             ts_point = TimeSeriesPoint(timestamp=date.timestamp(), value=count)
             formatted_data.append(ts_point)
     else:
```

Comment:
thoughts on decomposing this function a little? breaking out the contents of the `if` statements might be a good way to handle that (both just return their formatted data).